src/Arcus.Scripting.LogicApps/Arcus.Scripting.LogicApps.psm1

```diff
@@ -0,0 +1,23 @@
+<#
+ .Synopsis
+  Disable all specified Logic Apps described in the order control JSON file.
+  
+ .Description
+  Disables all specified Logic Apps in a specific order. The Logic Apps to be disabled and the order in which this will be done, will be defined in the deploy-OrderControl.json-file.
+
+ .Parameter DeployFileName
+  If your solution consists of multiple interfaces, you can specify the flow-specific name of the orderControl-file, if not, the script will look for a file named 'deploy-orderControl.json' by default.
+
+ .Parameter ResourceGroupName
+  The resource group containing the Azure Logic Apps.
```

Comment:
Let's switch the order, since ResourceGroupName is mandatory and DeployFileName is optional.

```suggestion
 .Parameter ResourceGroupName
  The resource group containing the Azure Logic Apps.
  
 .Parameter DeployFileName
  If your solution consists of multiple interfaces, you can specify the flow-specific name of the configuration file, if not, the script will look for a file named 'deploy-orderControl.json' by default.
```